internal/scraper/scraper.go

```diff
@@ -512,11 +512,23 @@ func (s Scraper) collectData(ctx context.Context, t time.Time) (*probeData, time
 
 	sl := kitlog.With(bl, loggerLabels...)
 
+	var timeout time.Duration
+	switch s.CheckType() {
+	case sm.CheckTypeMultiHttp, sm.CheckTypeScripted, sm.CheckTypeBrowser:
+		// k6-backed checks have a more sophisticated handling of timeouts, where the k6 runner receives the check
```

Comment:
I usually align everything to 120 chars as it is the "modern default", but don't have a problem changing it... provided we all agree on it :P

Since we'd need to change a whole lot of code it might be left for a follow-up though.